apm-agent-core/src/test/java/co/elastic/apm/agent/MockReporter.java

```diff
@@ -92,7 +92,7 @@
         when(apmServerClient.isAtLeast(any())).thenReturn(true);
         dslJsonSerializer = new DslJsonSerializer(mock(StacktraceConfiguration.class), apmServerClient);
         SPAN_TYPES_WITHOUT_ADDRESS = Set.of("jms");
-        SPAN_ACTIONS_WITHOUT_ADDRESS = Map.of("kafka", Set.of("poll"));
+        SPAN_ACTIONS_WITHOUT_ADDRESS = Map.of("kafka", Set.of("poll"), "rabbitmq", Set.of("send"));
```

Comment:
What functionality is impacted by that? Will RabbitMQ be visible as part of the service map?

What's preventing the capturing of the address? Maybe summarize the problems in a follow-up issue.

Reply:
> What functionality is impacted by that? Will RabbitMQ be visible as part of the service map?

For my side, I don't know if any feature is affected. I used the same solution that use the kafka plugin.

> What's preventing the capturing of the address? Maybe summarize the problems in a follow-up issue.

There isn't any attribute in the instrumented methods that allows obtain the address directly. I don't know if it can be obtained using other methods.

Reply:
> What functionality is impacted by that? Will RabbitMQ be visible as part of the service map?

The address is not required for the service map, only for the SIEM integration.

> There isn't any attribute in the instrumented methods that allows obtain the address directly. I don't know if it can be obtained using other methods.

If you know the address when the channel is created, you can store it in a map (channel object ---> address) and when the method is called, get it through the `@Advice.This` reference in the instrumented method.

Reply:
I see. Maybe one day I'll remember. Thanks for explaining again 😉 
So I would say it's not too critical at the moment.

Reply:
There isn't even a need to use a map in this case as we can use `Channel.getConnection()`, I've implemented that in the t commit.